app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,6 +2,9 @@ class Admin::DashboardController < ApplicationController
   before_action :authorize!
 
   def index
-
-  end  
+    @uncontacted_companies = Company.paginate(:page => params[:page],
+                                              :per_page => 5
+                                              ).uncontacted
+                                               .order("updated_at asc")
+  end
```

Comment:
Love the paginate! Should we add the order onto all of the status scopes so that it just always orders them by updated_at? I don't think that would be a problem anywhere else if it's always ordered by updated? 

  scope :contacted, -> { where(status: "contacted").order("updated_at asc")}
  scope :uncontacted, -> { where(status: "uncontacted").order("updated_at asc")}
  scope :confirmed, -> { where(status: "confirmed").order("updated_at asc")}
  scope :dead, -> { where(status: "dead").order("updated_at asc")}